make.xsh

```diff
@@ -492,6 +517,31 @@ def export_rootski_profile_aws_creds():
         )
 
 
+def export_rootski_profile_aws_creds_db_backup():
+    aws_creds_rel_path = ".aws/credentials"
```

Comment:
You. know what? I really like this function, but I think most of the logic duplicates the existing `export_rootski_profile_aws_creds()` function. Could you actually just call that function here? That would export the `AWS_SECRET_ACCESS_KEY` env vars. 

Then you could read those and re-export them with a new name using the `$` notation.

Reply:
Also, a docstring describing what it does would be very welcome. (unless `.xsh` files break when you add docstrings to pure functions, I don't think they do, but who knows). You can just say that it's exporting `DB_BACKUP__AWS_SECRET_ACCESS_KEY`, etc. by reading the `~/.aws/credentials` file.

Reply:
I created a single function that exports to AWS... instead of TRAEFIX_AWS... and BACKUP_DB_AWS...